spec/features/users/login_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe 'Log In' do 
+  describe 'A user visits the welcome page and attempts to log in' do
+    before :each do
+      @user = User.create!(email: 'me@email.com',
+                           password: 'password')
+    end
+
+    context 'with valid information' do 
+      before(:each) { visit '/' }
+
+      it 'I am a regular user, I log in and am redirected to my dashboard' do
+        fill_in 'email', with: "#{@user.email}"
+        fill_in 'password', with: "#{@user.password}"
+        click_button 'Sign In'
+
+        expect(current_path).to eq('/dashboard')
+        expect(page).to have_content("Welcome #{@user.email}!")
```

Comment:
Being picky here, but I think we only want to display the part of the email before the @